apps/www/registry/default/ui/breadcrumb.tsx

```diff
@@ -88,10 +82,10 @@ const BreadcrumbSeparator = ({
 )
 BreadcrumbSeparator.displayName = "BreadcrumbSeparator"
 
-const BreadcrumbEllipsis = ({
+const BreadcrumbEllipsis: React.FC<React.ComponentProps<"span">> = ({
   className,
   ...props
-}: React.ComponentProps<"span">) => (
+}) => (
```

Comment:
An example of a component that wasn't previously using `forwardRef` that I've updated to use `React.FC` for the sake of consistency.